package.json

```diff
@@ -13,7 +13,7 @@
 	"type": "module",
 	"exports": "./index.js",
 	"engines": {
-		"node": "^14.18.0 || ^16.14.0 || >=18.0.0"
+		"node": "^16.14.0 || >=18.0.0 || >=20.0.0"
```

Comment:
I am not sure the exact version range you'd prefer. Please let me know, and I can update it.